stages/org.osbuild.chmod

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/python3
+"""
+Change file mode bits
+
+Change the file mode bits of one or more files or directories inside the tree.
+"""
+
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util.path import in_tree
+
+SCHEMA = r"""
+"additionalProperties": false,
+"patternProperties": {
+  "^\\/(?!\\.\\.)((?!\\/\\.\\.\\/).)+$": {
```

Comment:
❤️
One thing though, wonder if we ever need global options for the stage and thus should move that to `items`, or `paths`?

Reply:
Top level "items" added.